.github/workflows/build.yml

```diff
@@ -171,7 +171,7 @@ jobs:
           # version 2.0.1; must be updated if the version changes
```

Comment:
My mistake, can you please update this comment to say 2.1.4?

Reply:
Done